### ISSUE TITLE
Hosted documentation

### ISSUE THREAD

**Issue by qunm00** [suchnsuch/Tangent#69]
Currently, documentation is available in the repo, but it's unavailable on the internet. It's difficult for end-user to view the documentation. tangentnotes.com also currently does not have link to documentation.

**Comment by qunm00** [comment]
GitHub Pages is a good choice.

**Comment by taylorhadden** [comment]
Yes, this is an issue that I've been sitting on for a long while. It's good to have it recorded.

GitHub Pages may be a decent stopgap choice, but I would really like to have a set of tools that makes it easy to publish any collection of Tangent-authored notes to the internet via SvelteKit.

**Comment by fredguth** [comment]
> GitHub Pages may be a decent stopgap choice, but I would really like to have a set of tools that makes it easy to publish any collection of Tangent-authored notes to the internet via SvelteKit.

How can this be accomplished? 
I would like to help with this issue.

**Comment by taylorhadden** [comment]
It will be a big lift. We'll want to separate out the note display/editing system from `tangent-electron` and into a module that can be shared. Ideally, we have some common Svelte components and infrastructure (like the workspace) that are used in both places.

I'm currently working on pulling the note parser out. The node editor can be a client of that module.

**Comment by taylorhadden** [comment]
Separating out the workspace & indexing system would definitely be part of this as well.

**Comment by fredguth** [comment]
One idea is to try to create an app `tangent-web` and moving everything that is common (workspace, views, etc) to a common place. Maybe in lib?

**Comment by taylorhadden** [comment]
That's a pretty reasonable starting point. Use "packages" instead though. "Lib" is for semi-external dependencies.
